clients/runner/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "runner"
-version = "1.0.10"
+version = "1.0.9"
```

Comment:
We shouldn't turn back the version.

Reply:
Sure! Must be as you say a faulty merge commit to main, because it is up-to-date now.